sexpress/src/main/scala/org/ensime/sexp/util/ThreadLocalSupport.scala

```diff
@@ -0,0 +1,7 @@
+package org.ensime.sexp.util
+
+trait ThreadLocalSupport {
```

Comment:
We do, just not in the parsers.  I didn't change this - just moved it out of the file that was deleted.